test/sample-data/test.compile.build.gradle.expected.js

```diff
@@ -0,0 +1,46 @@
+exports.expected = {
+    dependencies: [
+      {
+        group: 'junit',
+        name: 'junit',
+        version: '4.12',
+        type: 'testCompile',
+        excludes: []
+      },
+      {
+        group: 'org.codehaus.groovy',
+        name: 'groovy-all',
+        version: '2.4.10',
+        type: 'org',
+        excludes: []
+      },
+      {
+        group: 'org.jfrog.buildinfo',
+        name: 'build-info-extractor-gradle',
+        version: '4.5.4',
+        type: 'testCompile',
+        excludes: []
+      },
+      {
+        group: 'com.netflix.nebula',
+        name: 'gradle-lint-plugin',
+        version: 'latest.release',
+        type: 'com',
+        excludes: []
+      },
+      {
+        group: '',
+        name: ' \n',
```

Comment:
Hmm, this group sure seems like a bug - any idea what the deal is here? :-) @dilipkumar2k6

Reply:
Need to look into it. Yaa, seems like a bug but not sure.

Reply:
I found that, in following declaration, if I keep closing bracket `]` at the new line then it generates `name: ' \n'`. 

Generates new line as output: 
```
  testCompile([
      'org.jfrog.buildinfo:build-info-extractor-gradle:4.5.4',
      'com.netflix.nebula:gradle-lint-plugin:latest.release'
      ]
      )
```
It works fine: 
```
  testCompile([
      'org.jfrog.buildinfo:build-info-extractor-gradle:4.5.4',
      'com.netflix.nebula:gradle-lint-plugin:latest.release' ]
      )
```
I think this is separate issue and can be tracked separately.

Reply:
Added #24 based on your comment. :-)